SabreTools.Models/N3DS/CIA.cs

```diff
@@ -14,6 +14,11 @@ namespace SabreTools.Models.N3DS
     /// <see href="https://www.3dbrew.org/wiki/CIA"/>
     public class CIA
     {
+        /// <summary>
+        /// Offset records for each file in the CIA
+        /// </summary>
+        public List<CIAOffsets>? Offsets { get; set; }
```

Comment:
Please put all model fields in order of where they appear in the file. If this isn't something that is actually in the file as a table or array, then consider having this be on the Wrapper implementation later.